modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue

```diff
@@ -84,6 +84,14 @@ export default {
             label: 'Page Title',
             name: 'title'
           },
+          {
+            label: 'Published',
+            name: 'lastPublishedAt',
+            value: {
+              true: '',
+              false: 'Unpublished'
```

Comment:
Maybe this should be `indicator`? `value` is more of a computer-read property. I realize `label` is taken here.

Reply:
yeah value didn't seem right to me .. what about `label` being able to be both an object and a string and figuring it out down there at the template?

Reply:
This is describing the column. `columnHeader` could be appropriate, [based on the related aria-role name](https://w3c.github.io/aria/#columnheader). Then `label` is freed up. Maybe it'd be good to specify both better: `columnHeader` or `header` and then `cellLabel`.

Reply:
oo that would not really be that same thing ... maybe `label` should change to `heading` to be more in line with how a table gets constructed?

then i could take `label` for this

Reply:
Simply making `label` do more work doesn't seem right.

Reply:
fine w the explicit versions of `columnHeader` and `cellLabel`

Reply:
I like the improved property names.